### DISCUSSION TITLE
Average Response Time of an Agent in Report

### DISCUSSION THREAD

**Discussion by hosanak** [chatwoot/chatwoot#4739] [Feature Requests]
hi all,

I have been trying to find in the internet, in this github, and also in the documentation, but there is only "First Response Time". I could not find the statistics of "Average Response Time" of an agent. Wondering whether it is possible to have this feature?

Thanks.

**Comment by sojan-official** [comment]
@hosanak at the moment we don't track it as a metric.  We could look into it in the future if there is enough interest in the community.

**Comment by sojan-official** [comment]
ref: https://github.com/chatwoot/chatwoot/issues/4744
